rasa/nlu/training_data/message.py

```diff
@@ -95,11 +95,14 @@ def __hash__(self) -> int:
         return hash((self.text, str(ordered(self.data))))
 
     @classmethod
-    def build(cls, text, intent=None, entities=None, **kwargs) -> "Message":
+    def build(
+        cls, text: Text, intent: Optional[Text] = None, entities=None, **kwargs
```

Comment:
Should we add the type for `entities` as well?

Reply:
yes sure, tried to look it up but couldn't. Is this a list of strings? or something else?

Reply:
Not completely sure. @tabergma can you help here?

Reply:
entities should be a `List[Dict[Text, Any]]`